tests/unit/test_util_types.py

```diff
@@ -287,3 +287,23 @@ def test_version_compare(self):
         self.assertTrue(Version('4.0-SNAPSHOT2') > Version('4.0.0-SNAPSHOT1'))
 
         self.assertTrue(Version('4.0.0-alpha1-SNAPSHOT') > Version('4.0.0-SNAPSHOT'))
+
+
+class Methods(unittest.TestCase):
```

Comment:
class name should end with `Test` or `Tests`

regardless the name is kind of vague

Reply:
Can you suggest a name for a suit for all the methods in the module ?

Reply:
If it doesn't have Test I the class name

Pytest won't select it

Reply:
Fixed.